pkg/activator/throttler_test.go

```diff
@@ -256,7 +256,12 @@ func TestThrottlerTry(t *testing.T) {
 				initCapacity)
 
 			if s.addCapacity {
-				throttler.UpdateCapacity(revID, 1)
+				// If we had success updating the capacity > 0, check that it's properly returned.
+				if throttler.UpdateCapacity(revID, 1) == nil {
+					if got, want := throttler.GetRevisionCapacity(revID), 1*10; got != want {
```

Comment:
Let's externalize the `10` as a constant at the top of the test saying `containerConcurrency`. That makes the whole thing a bit clearer I think. Maybe also externalize `numPods` or something on top of this if to make it speaking throughout.

Reply:
Did most of them.
I kept 1, since it's local to the test.